backend/sirius-web-interpreter/src/test/java/org/eclipse/sirius/web/interpreter/AQLInterpreterTestCases.java

```diff
@@ -92,4 +92,20 @@ public void testBrokenEPackage() {
         assertThat(interpreter).isNotNull();
     }
 
+    @Test
+    public void testNullExpressionEvaluatesToTrue() {
+        AQLInterpreter interpreter = new AQLInterpreter(List.of(), List.of(EcorePackage.eINSTANCE));
+        Result result = interpreter.evaluateExpression(Map.of(), null);
+        assertThat(result).isNotNull();
+        assertThat(result.asBoolean()).contains(Boolean.TRUE);
+    }
+
+    @Test
+    public void testEmptyExpressionEvaluatesToTrue() {
+        AQLInterpreter interpreter = new AQLInterpreter(List.of(), List.of(EcorePackage.eINSTANCE));
+        Result result = interpreter.evaluateExpression(Map.of(), ""); //$NON-NLS-1$
+        assertThat(result).isNotNull();
+        assertThat(result.asBoolean()).contains(Boolean.TRUE);
+    }
+
```

Comment:
Just to be sure, is that the default behavior in Sirius RCP?

Reply:
Good remark, it's actually the opposite. I'll change the new tests and update the actual code to match (hoping this does not have too many impacts).

Reply:
Actually... I was right the first time :) I actually tested the behavior of Sirius Desktop, and either `aql:''` or `aql:'  '` (non-empty but blank strings) are true when internepreted as booleans. Our code in `org.eclipse.sirius.web.interpreter.Result.asBoolean()` it directly taken from https://github.com/pcdavid/org.eclipse.sirius/blob/master/plugins/org.eclipse.sirius.common/src/org/eclipse/sirius/common/tools/internal/interpreter/DefaultConverter.java#L36